var/spack/repos/builtin/packages/ldak/package.py

```diff
@@ -33,8 +33,8 @@ class Ldak(Package):
 
     requires("target=x86_64:", when="~glpk", msg="bundled qsopt is only for x86_64")
     requires(
-        "^mkl",
```

Comment:
I could have sworn I tested this when I made it. Oh well.

Reply:
It used to work